Gemfile

```diff
@@ -11,7 +11,14 @@ else
   gem "rails", "~> #{rails_version}"
 end
 
-gem "rack-test"
+group :test do
+  if RUBY_VERSION == "1.9.3"
```

Comment:
Cool. I was hoping the duplicate version wouldn't be an issue. Multiple Gemfiles is an option, but... 

What we should do here really, is bump the gem version to the next minor, `0.2.0`. What we can do then is announce breaking changes by dropping support for 1.9.3. That way, you don't have to do the multiple Gemfile dance.